### DISCUSSION TITLE
Feature Request:Make Laragon For linux

### DISCUSSION THREAD

**Discussion by Khant-Nyar** [leokhoa/laragon#513] [Q&A]
I love to use Laragon,It is easy to use and awesome feature so I am also want to use in my linux machine.It possible in the future?.

**Comment by bishwajitcadhikary** [comment]
There already an alternative is avaliable [Valet Linux + ](https://valetlinux.plus)

**Reply by Khant-Nyar** [reply]
Not Match guys, i want to change php version ,composer version, etc...,but i am using kali so kali don't provide for php old version 7.4.x and also can't add third party repositories

**Reply by bishwajitcadhikary** [reply]
composer self-update 1.10.22
composer self-update 2.1.3

sudo add-apt-repository ppa:ondrej/php
use this repository

valet use 8.1
valet use 8.2
valet use 7.4

**Comment by 3xploiton3** [comment]
i wish i could run laragon in arch linux, i love how laragon works in windows 11
